src/components/Timer/Timer.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useRef } from 'react';
+import TimerButton from '../Features/TimerButton';
+import { formatTime } from '../Features/Formater';
+
+const Timer: React.FunctionComponent = () => {
```

Comment:
1. Don't import react
2. 2Dont use React.FunctionCOmponent - import FUnctionComponent from React and just write TImer: FunctionComponent